assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+          },
+          "freeform": {
+            "key": "capacity",
+            "type": "pnat"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access"
+          },
+          "question": {
+            "en": "Does this place provide internet access?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "internet_access=yes"
+                ]
+              },
+              "then": {
+                "en": "There is internet access"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "internet_access=no"
+                ]
+              },
+              "then": {
+                "en": "There is no internet access"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "internet_access"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access price"
+          },
+          "question": {
+            "en": "Do you have to pay for the internet access?"
+          },
+          "freeform": {
+            "key": ""
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "internet_access:fee=yes"
+                ]
+              },
+              "then": {
+                "en": "You need to pay extra for internet access"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "internet_access:fee=no"
+                ]
+              },
+              "then": {
+                "en": "You  do not need to pay extra for internet access"
+              }
+            }
+          ],
+          "condition": {
+            "and": [
+              "internet_access=yes"
+            ]
+          }
+        },
+        "reviews",
+        {
+          "render": {
+            "en": "toilets"
+          },
+          "question": {
+            "en": "Does this place have toilets?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "toilets=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has toilets"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "toilets=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have toilets"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "toilets"
+          }
+        },
+        {
+          "render": {
+            "en": "More details about this place: {description}"
+          },
+          "question": {
+            "en": "Would you like to add a general description of this place? (please keep it objective; opinions go into the reviews)"
```

Comment:
I have noticed that people tend to repeat information that was already stated in the earlier asked questions; it doesn't hurt to state something like "Do not repeat information previously asked or shown above"

Reply:
done